libvcx/src/proof.rs

```diff
@@ -1086,104 +1044,26 @@ pub mod tests {
         assert_eq!(release(h5).unwrap_err().kind(), VcxErrorKind::InvalidProofHandle);
     }
 
-    #[ignore]
     #[test]
     #[cfg(feature = "general_test")]
     fn test_proof_validation_with_predicate() {
-        let _setup = SetupLibraryWallet::init();
-
-        pool::tests::open_test_pool();
-        //Generated proof from a script using libindy's python wrapper
-
-        let proof_msg: ProofMessage = serde_json::from_str(PROOF_LIBINDY).unwrap();
-        let mut proof_req_msg = ProofRequestMessage::create();
-        proof_req_msg.proof_request_data = serde_json::from_str(PROOF_REQUEST).unwrap();
-        let mut proof = Proof {
-            source_id: "12".to_string(),
-            msg_uid: String::from("1234"),
-            ref_msg_id: String::new(),
-            requested_attrs: String::from("[]"),
-            requested_predicates: REQUESTED_PREDICATES.to_string(),
-            state: VcxStateType::VcxStateRequestReceived,
-            proof_state: ProofStateType::ProofUndefined,
-            name: String::new(),
-            version: String::from("1.0"),
-            nonce: generate_nonce().unwrap(),
-            my_did: None,
-            my_vk: None,
-            their_did: None,
-            their_vk: None,
-            agent_did: None,
-            agent_vk: None,
-            proof: Some(proof_msg),
-            proof_request: Some(proof_req_msg),
-            revocation_interval: RevocationInterval { from: None, to: None },
-            thread: Some(Thread::new()),
-        };
-        apply_agent_info(&mut proof, &default_agent_info(None));
-
-        let rc = proof.proof_validation();
-        assert!(rc.is_ok());
-        assert_eq!(proof.proof_state, ProofStateType::ProofValidated);
-
-        let proof_data = proof.get_proof().unwrap();
-        assert!(proof_data.contains(r#""schema_seq_no":694,"issuer_did":"DunkM3x1y7S4ECgSL4Wkru","credential_uuid":"claim::1f927d68-8905-4188-afd6-374b93202802","attr_info":{"name":"age","value":18,"type":"predicate","predicate_type":"GE"}}"#));
-    }
-
-    #[ignore]
-    #[test]
-    #[cfg(feature = "general_test")]
-    fn test_send_proof_request_can_be_retried() {
```

Comment:
Let's keep this one and just keep it ignored if it's problematic to restore for aries.